src/schema/artist/index.js

```diff
@@ -428,45 +428,41 @@ export const ArtistType = new GraphQLObjectType({
             partner_shows: numeral(
               ({ partner_shows_count }) => partner_shows_count
             ),
-            related_artists: numeral(({ related_artists }) => related_artists),
-            articles: numeral(({ articles }) => articles),
+            related_artists: {
+              type: GraphQLInt,
+              resolve: (
+                { id },
+                _options,
+                _request,
+                { rootValue: { relatedMainArtistsLoader } }
+              ) => {
+                return totalViaLoader(
+                  relatedMainArtistsLoader,
+                  {},
+                  {
+                    artist: [id],
+                  }
+                )
+              },
+            },
+            articles: {
+              type: GraphQLInt,
+              resolve: (
+                { _id },
+                _options,
+                _request,
+                { rootValue: { articlesLoader } }
+              ) =>
+                articlesLoader({
```

Comment:
ah so any `{ counts {...} }` selection would make a call to `/articles`, got it makes sense